README.md

```diff
@@ -5,17 +5,19 @@ bringing you reproducible builds and better caching.
 
 This repository hosts two components:
 
-- a [Nixpkgs](https://github.com/NixOS/nixpkgs) overlay, located at `overlay`
+- A [nixpkgs](https://github.com/NixOS/nixpkgs) overlay, located at the `/overlay`
```

Comment:
The name "nixpkgs" should be capitalized for consistency with the official Nix documentation.

Reply:
Done :)